CHANGELOG.md

```diff
@@ -7,6 +7,19 @@ and this project does **not** adhere to [Semantic Versioning](https://semver.org
 
 ## [Unreleased]
 
+### Added
+
+-   The popup of a simulated region now has gut three tabs: _Overview_, _Transfers_ and _Behaviors_
```

Comment:
What does `gut` mean? :P

Reply:
Was a typo, should have been "got" 😅.

Fixed it in https://github.com/hpi-sam/digital-fuesim-manv/pull/724/commits/47dd9ec520a7a29b40dd3ee2ee8be4e165cb95be.